### PR TITLE
Improve wording for external crate resolution error

### PR DESCRIPTION
**Pull request by camelid**
I think it reads better this way.

### CODE REVIEW

**Review by davidtwco — APPROVED** [review]
LGTM, r=me when CI passes

File: compiler/rustc_resolve/src/imports.rs [hunk 1]
```diff
@@ -1157,7 +1157,7 @@ impl<'a, 'b> ImportResolver<'a, 'b> {
                     }
                     _ => {
                         if !ident.is_path_segment_keyword() {
-                            format!("no `{}` external crate", ident)
+                            format!("no external crate `{}`", ident)
```
Comment (camelid) [inline compiler/rustc_resolve/src/imports.rs:1160]: I considered changing it to "no external crate **named**", but I didn't want to be wordy. What do you think?

```suggestion
                            format!("no external crate named `{}`", ident)
```

Comment (davidtwco) [reply compiler/rustc_resolve/src/imports.rs:1160]: I think that the PR's current wording is fine, but I don't feel strongly - either is an improvement.

**Comment by camelid** [comment]
Oh well, worth a try :)

**Pull request merged**